Makefile

```diff
@@ -10,41 +10,48 @@ help:
 	@echo '     make clean.container shutdown running container and delete image'
 	@echo '     make clean.ws        delete all groovy/gradle artifacts in the workspace'
 	@echo '     make build           build a dockerfile for testing the jenkins configuration'
-	@echo '     make run             run the dockerfile in the background'
+	@echo '     make run             run the dockerfile and kickoff jenkins'
+	@echo '     make run.container   run the dockerfile without starting the jenkins .war'
+	@echo '     make run.jenkins     run the jenkins .war on the container'
 	@echo '     make logs            tail the logs for the Jenkins container'
```

Comment:
GH won't let me comment on the right line (because it's not part of this change set) but could you mark these targets as PHONY?